lib/src/exceptions/exceptions.dart

```diff
@@ -99,6 +99,8 @@ abstract class SoLoudCppException extends SoLoudException {
         return const SoLoudFilterAlreadyAddedException();
       case PlayerErrors.playerAlreadyInited:
         return const SoLoudPlayerAlreadyInitializedException();
+      case PlayerErrors.soundHandleNotFound:
+        return const SoLoudSoundHashNotFoundCppException();
```

Comment:
It's confusing to have "soundhandle" error create a "soundhash" exception. We should pick one.

Reply:
Ouch, my bad. Thank you!